pkg/devfile/adapters/common/utils_test.go

```diff
@@ -533,7 +536,11 @@ func TestGetCommands(t *testing.T) {
 				},
 			}
 
-			commandsMap := devObj.Data.GetCommands()
+			commandsMap, err := devObj.Data.GetCommands()
```

Comment:
not sure what this test is testing, this looks like redundant if it is already being tested at the parser level but now i feel we should test one err condition if this is going to stay here